src/store/actions.js

```diff
@@ -3,17 +3,26 @@ export function someAction (context) {
 }
 */
 import axios from "axios";
-export function getPosts({ commit }, rssUrl) {
-  const rssArray = rssUrl.replace(/ /g, "").split(",");
-  rssArray.forEach(rss => {
+export function getPosts({ commit }, rss) {
+
+  const rssArray = rss.split(",");
+  const rssUrlsWithOptionsArray = rssArray.map(rss => {
+    const rssUrlWithOptions = rss.split(" ");
+    return {
+      rssUrl: rssUrlWithOptions[0],
+      rssItemsCount: rssUrlWithOptions[1]
```

Comment:
We should check this is a number or set to null.

Reply:
Please see if current implementation looks good!